sagemaker/launch/launch_partition.py

```diff
@@ -78,21 +60,21 @@ def run_job(input_args, image):
         debugger_hook_config=False,
         entry_point=os.path.basename(entry_point),
         source_dir=os.path.dirname(entry_point),
-        image_uri=container_image_uri,
+        image_uri=image,
         role=role,
         instance_count=instance_count,
         instance_type=instance_type,
         py_version="py3",
         hyperparameters=params,
         sagemaker_session=sagemaker_session,
-        base_job_name=f"gs-partition-{graph_name}",
+        base_job_name=f"gs-partition-{sm_task_name}",
```

Comment:
if task name is None, can we give a default name instead of ""?

Reply:
Sure, I'll replace it with a timestamp